setupTests.js

```diff
@@ -3,3 +3,15 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 window.HTMLElement.prototype.scrollIntoView = () => {};
+
+/* ---------------------------------------------------------------------------------------------
+ * Below is hack to allow testing of PointerEvent as used in github package
+ * https://github.com/pmndrs/use-gesture
+ ---------------------------------------------------------------------------------------------*/
+/* eslint-disable no-console */
```

Comment:
Remove this (should be not needed?) and the reference from github.. now we have nothing in here

Reply:
removed